tensorflow-examples/src/main/java/org/tensorflow/model/examples/cnn/alexnet/AlexNetModel.java

```diff
@@ -0,0 +1,294 @@
+ */
+public class AlexNetModel implements AutoCloseable {
+    private static final int PIXEL_DEPTH = 255;
+    private static final int NUM_CHANNELS = 1;
+    private static final int IMAGE_SIZE = 28;
+    private static final int NUM_LABELS = 26;
+    private static final long SEED = 123456789L;
+
+    private static final String PADDING_TYPE = "SAME";
+    public static final String INPUT_NAME = "input";
+    public static final String OUTPUT_NAME = "output";
+    public static final String TARGET = "target";
+    public static final String TRAIN = "train";
+    public static final String TRAINING_LOSS = "training_loss";
+    public static final String INIT = "init";
+
+    private static final Logger logger = Logger.getLogger(AlexNetModel.class.getName());
+
+    private final Graph graph;
+
+    private final Session session;
+
+    public AlexNetModel() {
+        graph = compile();
+        session = new Session(graph);
+    }
+
+    public static Graph compile() {
+        Graph graph = new Graph();
+
+        Ops tf = Ops.create(graph);
+
+        // Inputs
+        Placeholder<TUint8> input = tf.withName(INPUT_NAME).placeholder(TUint8.DTYPE,
+                Placeholder.shape(Shape.of(-1, IMAGE_SIZE, IMAGE_SIZE)));
+        Reshape<TUint8> input_reshaped = tf
+                .reshape(input, tf.array(-1, IMAGE_SIZE, IMAGE_SIZE, NUM_CHANNELS));
+        Placeholder<TUint8> labels = tf.withName(TARGET).placeholder(TUint8.DTYPE);
+
+        // Scaling the features
+        Constant<TFloat32> centeringFactor = tf.constant(PIXEL_DEPTH / 2.0f);
+        Constant<TFloat32> scalingFactor = tf.constant((float) PIXEL_DEPTH);
+        Operand<TFloat32> scaledInput = tf.math
+                .div(tf.math.sub(tf.dtypes.cast(input_reshaped, TFloat32.DTYPE), centeringFactor),
+                        scalingFactor);
+
+        //Layer 1
+        Relu<TFloat32> relu1 = alexNetConv2DLayer("1", tf, scaledInput, new int[]{11, 11, NUM_CHANNELS, 96}, 96);
+        MaxPool<TFloat32> pool1 = alexNetMaxPool(tf, relu1);
+        LocalResponseNormalization<TFloat32> norm1 = alexNetModelLRN(tf, pool1);
+
+        //Layer 2
+        Relu<TFloat32> relu2 = alexNetConv2DLayer("2", tf, norm1, new int[]{5, 5, 96, 256}, 256);
+        MaxPool<TFloat32> pool2 = alexNetMaxPool(tf, relu2);
+        LocalResponseNormalization<TFloat32> norm2 = alexNetModelLRN(tf, pool2);
+
+        //Layer 3
+        Relu<TFloat32> relu3 = alexNetConv2DLayer("3", tf, norm2, new int[]{3, 3, 256, 384}, 384);
+        LocalResponseNormalization<TFloat32> norm3 = alexNetModelLRN(tf, relu3);
+
+        //Layer 4
+        Relu<TFloat32> relu4 = alexNetConv2DLayer("4", tf, norm3, new int[]{3, 3, 384, 384}, 384);
+
+        //Layer 5
+        Relu<TFloat32> relu5 = alexNetConv2DLayer("2", tf, relu4, new int[]{3, 3, 384, 256}, 256);
+        MaxPool<TFloat32> pool5 = alexNetMaxPool(tf, relu5);
+        LocalResponseNormalization<TFloat32> norm5 = alexNetModelLRN(tf, pool5);
```

Comment:
Dear @akshaybahadur21 could you please explain, why are you using LRN here (please share the link on AlexNet references)?